.github/workflows/aws-prod.yml

```diff
@@ -0,0 +1,67 @@
+on:
+  push:
+    branches:
+      - master
+
+name: Deploy Teleport Keeper into Production ECS
+
+jobs:
+  deploy:
+    name: Deploy
+    runs-on: ubuntu-latest
+    env:
+      REGION: us-east-1
+      SERVICE_NAME: teleport-keeper-prod
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Configure AWS credentials
+      uses: aws-actions/configure-aws-credentials@v1
+      with:
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        aws-region: ${{ env.REGION }}
+
+    - name: Login to container registry
+      id: login-ecr
+      uses: aws-actions/amazon-ecr-login@v1
+
+    - name: Build, tag, and push image to container registry
+      id: build-image
+      env:
+        ECR_REGISTRY: ${{ steps.login-ecr.outputs.registry }}
+        IMAGE_TAG: latest
+      run: |
+        # Build a docker container and
+        # push it to ECR so that it can
+        # be deployed to ECS.
+        git submodule update --init --recursive
```

Comment:
@nickkounz this shouldn't be needed if you're checking out the repo above

Reply:
Submodules need to be pulled, IF `.gitmodules` are defined. As mentioned by Cristi, this can be done right in the checkout repo module, ex: https://github.com/makerdao/maker-keeper/blob/49fb0b800193dbcc4e96dd944e715e072186fa30/.github/workflows/publish.yaml#L16